### PR TITLE
Engdocs 1318

### PR DESCRIPTION
**Pull request by aevesdocker**
Company self-serve content

### CODE REVIEW

File: docker-hub/new-company.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: Create a company
+description: Learn how to create a company.
+keywords: company, hub, organization, company owner
+---
+
+## Create a company
+
+To create a new company:
+
+1. In Docker Hub, navigate to an organization you want to place under a company. The organization must have a Business subscription.
```
Comment (chrstnfrrs) [inline docker-hub/new-company.md:11]: Could something like this sound more natural?

```suggestion
1. In Docker Hub, navigate to an organization on a business subscription that want to place under a company.
```

**Comment by mercedessteeledockertest** [comment]
Looking great! 
One thing I noticed: https://deploy-preview-17189--docsdocker.netlify.app/docker-hub/new-company/
organisations and organizations are used

File: docker-hub/new-company.md (hunk 1 above)
Comment (aevesdocker) [reply docker-hub/new-company.md:11]: I personally think it's ok. I think it reads clearer if it is two sentences. This PR will be checked by my docs-team colleagues and if they pick up on it too, I'll reassess.

File: docker-hub/new-company.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+2. Select **Settings**.
+3. Near the bottom of the **General** tab, select **Create a company**.
+4. Enter a unique name for your company.
```
[the remark this answers is not in the document]
Comment (aevesdocker) [reply docker-hub/new-company.md:14]: Yes, good catch! Amending now.

**Pull request merged**